tools/test/examples/examples_cmake.json

```diff
@@ -0,0 +1,150 @@
+{
+  "examples": [
+    {
+        "name": "mbed-os-example-blinky",
+        "github": "https://github.com/ARMmbed/mbed-os-example-blinky",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : [],
```

Comment:
Should you specify an Mbed target here?

Reply:
`[ ]` = all supported target, that was in the README https://github.com/ARMmbed/mbed-os/tree/master/tools/test/examples#values
It is counter-intuitive, I'd say. but we are not going to change it in the PR now